covasim/webapp/cova_app.css

```diff
@@ -108,7 +108,7 @@ fieldset > legend {
     grid-area: content;
     margin: 0;
     display: grid;
-    grid-template-columns: 3fr 4fr;
+    grid-template-columns: 2fr 4fr;
```

Comment:
Would be good for this to be more responsive, but on my screen at least 3fr was too wide